Loop/Managers/LoopDataManager.swift

```diff
@@ -491,7 +489,8 @@ extension LoopDataManager {
                 if error == nil {
                     self.insulinEffect = nil
                     // Expire any bolus values now represented in the insulin data
-                    if let bolusDate = self.lastRequestedBolus?.date, bolusDate.timeIntervalSinceNow < TimeInterval(minutes: -5) {
+                    // TODO: Ask pumpManager if dose represented in data
```

Comment:
I think we should eventually make pumpmanagers 100% responsible for dose finalization, since that process varies based on pump implementation (whether it has history, or other controllers, etc)